src/program.js

```diff
@@ -119,9 +119,14 @@ export class Program {
 
 
 export function defaultVersionGetter(absolutePackageDir: string): string {
-  let packageData: any = readFileSync(
+  if (process.env.NODE_ENV === 'production') {
+    let packageData: any = readFileSync(
```

Comment:
let packageData: string = readFileSync(
                           ^ Buffer. This type is incompatible with string